config/initializers/braintree.rb

```diff
@@ -0,0 +1,11 @@
+if Rails.env.production?
+  Braintree::Configuration.environment = :production
+  Braintree::Configuration.merchant_id = ENV['BRAINTREE_MERCHANT_ID']
+  Braintree::Configuration.public_key = ENV['BRAINTREE_PUBLIC_KEY']
+  Braintree::Configuration.private_key = ENV['BRAINTREE_PRIVATE_KEY']
+else
+  Braintree::Configuration.environment = :sandbox
+  Braintree::Configuration.merchant_id = ENV['BRAINTREE_MERCHANT_ID']
+  Braintree::Configuration.public_key = ENV['BRAINTREE_PUBLIC_KEY']
+  Braintree::Configuration.private_key = ENV['BRAINTREE_PRIVATE_KEY']
+end
```

Comment:
Setting `merchant_id`, `public_key`, and `private_key` can be done outside of the `if` block.